tests/pipeline/test_mpca_trainer.py

```diff
@@ -5,8 +5,11 @@
 from sklearn.metrics import accuracy_score, roc_auc_score
 
 from kale.pipeline.mpca_trainer import MPCATrainer
+from kale.utils.download import download_file_by_url
 
-gait = loadmat("tests/test_data/gait_gallery_data.mat")
+gait_url = "https://github.com/pykale/data/raw/main/video_data/gait/gait_gallery_data.mat"
+download_file_by_url(gait_url, "../test_data", "gait.mat", "mat")
+gait = loadmat("../test_data/gait.mat")
```

Comment:
Line 10 to 12 has been _repeated_ three times in this PR. You should move it to `conftest.py` and then use `gait` directly, like the way we reuse `download_path`.